perlin.go

```diff
@@ -18,9 +18,11 @@ const (
 
 // Perlin is the noise generator
 type Perlin struct {
-	alpha float64
-	beta  float64
-	n     int
+	alpha  float64
+	beta   float64
+	n      int
+	source rand.Source
```

Comment:
Well you need at least the Rand because you call Int() on that instance of it. I suppose the Source could be dropped. Would you like me to remove it?